src/components/Notifications/RuleSimple/RuleSimple.js

```diff
@@ -196,6 +206,12 @@ export default class RuleSimple extends React.Component {
         session_filter_locks: lockedFilters,
       },
     ]
+    const selectedNumberColumnName = this.state.selectedColumns.map(
+      (index) => this.props.queryResponse?.data?.data?.columns[index]?.name,
+    )[0]
+    if (this.NUMBER_COLUMNS_AMOUNT >= 2) {
```

Comment:
Looking at this I am thinking we should have a condition to exclude the groupables from this number columns amount variable. There is a case where there could be 2 number columns, but one of them is a groupable, so really only 1 number column would be selectable